app/scripts/directives/showErrors.js

```diff
@@ -20,7 +17,7 @@ angular.module('confRegistrationWebApp').directive('showErrors', function () {
             ngModelCtrl.$invalid &&
             (scope.currentPageVisited ||
               ngModelCtrl.$touched ||
-              !!scope.showErrorsInstant)
+              !!scope.$eval(attrs.showErrorsInstant))
```

Comment:
Before, it used `!!attrs.showErrorsInstant`. I'm curious why it needed to change to `!!scope.$eval(attrs.showErrorsInstant)`. Is something processing differently on edit?

Reply:
Before, any non-empty value was interpreted as `true`. However, I needed `show-errors-instant` to be able to be an expression for this line https://github.com/CruGlobal/conf-registration-web/blob/d22b81295a235aca70d8989780864113dc054b53/app/views/blocks/emailQuestion.html#L3 so that previously saved invalid emails would be immediately marked as errors, as requested by Ben.